src/components/MemoryItem.stories.js

```diff
@@ -0,0 +1,15 @@
+import MemoryItem from './MemoryItem'
+import image from '../assets/images/memory_1.jpg'
```

Comment:
Probably a more declarative name would be a bit better even though just image works aswell of course

Reply:
Changed